src/assets/css/font-awesome.css

```diff
@@ -6,8 +6,7 @@
  * -------------------------- */
 @font-face {
   font-family: 'FontAwesome';
-  src: url('../fonts/fontawesome-webfont.eot?v=4.7.0');
-  src: url('../fonts/fontawesome-webfont.eot?#iefix&v=4.7.0') format('embedded-opentype'), url('../fonts/fontawesome-webfont.woff2?v=4.7.0') format('woff2'), url('../fonts/fontawesome-webfont.woff?v=4.7.0') format('woff'), url('../fonts/fontawesome-webfont.ttf?v=4.7.0') format('truetype'), url('../fonts/fontawesome-webfont.svg?v=4.7.0#fontawesomeregular') format('svg');
+  src: url("chrome-extension://__MSG_@@extension_id__/assets/fonts/fontawesome-webfont.woff2");
```

Comment:
The **font-face** rule allows custom fonts to be loaded on the webpage. Once added the stylesheet, the rule instructs the browser to get the font from where it is hosted ( in this case, tries to get the fonts from **uscourts.gov** and return a 404 code), then display it as specified in the CSS.

This fix will change the source of the font-face rule to use the files included in the asset folder of the extension.

We can use another approach to avoid changing the assets we get from Font Awesome. We can use the style.css file to create a new **font-face** rule that uses the files in the asset folder and then overwrite the .fa class to use the newly created font. I think this approach will work event if we upgrade our version of Font Awesome.

Reply:
Oh, that makes sense, thank you. Yes, if you can make that other approach work, that's great.